app/src/main/java/com/example/coronadiagnosticapp/ui/views/FeelingQuestionView.kt

```diff
@@ -40,10 +40,20 @@ class FeelingQuestionView : LinearLayout {
     private fun addFeelings() = GeneralFeeling.values()
         .forEach { addFeeling(it) }
 
+    private fun getLocalizedFeeling(feeling: String): CharSequence? {
+        var res = ""
+        when (feeling) {
+            "Same" -> res = context.getString(R.string.same)
+            "Better" -> res = context.getString(R.string.better)
+            "Worse" -> res = context.getString(R.string.worse)
+        }
+        return res
+    }
+
     private fun addFeeling(feeling: GeneralFeeling) {
         val btn = EkRadioBtn(context).apply {
             id = feeling.ordinal
-            text = feeling.name.toLowerCase().capitalize()
+            text = getLocalizedFeeling(feeling.name.toLowerCase().capitalize())
```

Comment:
Is it working and saving data in English when you are on Heb and Eng ? 
and maybe you should create more generic method so when a user rather if the system on ENG or HEB 
do something which required saving data to the DB, you will get the data, 'translate' it to eng and only than save it to the db

Reply:
Yes, 100% 
The db needs to save English enum values